Lib/fontTools/ttLib/tables/_g_v_a_r.py

```diff
@@ -37,46 +38,18 @@
 
 GVAR_HEADER_SIZE = sstruct.calcsize(GVAR_HEADER_FORMAT)
 
-class _lazy_dict(dict):
+class _LazyDict(UserDict):
```

Comment:
maybe we could move this `_LazyDict` to `fontTools.misc.dictTools` or something, so can be used elsewhere.. Or maybe we do that if/when we find a use for it.

Reply:
I think for now it's fine here.